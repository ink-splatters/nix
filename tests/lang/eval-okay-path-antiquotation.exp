{ absolute = /foo; expr = /home/eelco/Dev/nix/tests/lang/foo/bar; home = /tmp/nix-shell.ZeLfDw/nix-test/default/test-home/foo; notfirst = /home/eelco/Dev/nix/tests/lang/bar/foo; simple = /home/eelco/Dev/nix/tests/lang/foo; slashes = /foo/bar; surrounded = /home/eelco/Dev/nix/tests/lang/a-foo-b; }
