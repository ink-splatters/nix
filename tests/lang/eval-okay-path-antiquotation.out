{ absolute = /foo; expr = /home/bmc/sources/nix/tests/lang/foo/bar; home = /fake-home/foo; notfirst = /home/bmc/sources/nix/tests/lang/bar/foo; simple = /home/bmc/sources/nix/tests/lang/foo; slashes = /foo/bar; surrounded = /home/bmc/sources/nix/tests/lang/a-foo-b; }
